ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -107,15 +110,15 @@
 
 (defmethod field :string [{:keys [update! label name max-length min-length regex
                                   focus on-blur form? error warning table? full-width?
-                                  style input-style hint-style password? on-enter]
+                                  style input-style hint-style password? on-enter hint-text]
                            :as   field} data]
   [text-field
    (merge
     {:name name
      :floating-label-text (when-not table? label)
      :floating-label-fixed true
      :on-blur           on-blur
-     :hintText          (placeholder field data)
+     :hint-text         (if hint-text hint-text "")
```

Comment:
Is `:placeholder` support removed? It is used in service-search filters at least.

Also `(or hint-text "")` is more idiomatic.